platform/ui-next/src/components/SidePanel/SidePanel.tsx

```diff
@@ -143,47 +139,67 @@ const createBaseStyle = (expandedWidth: number) => {
     height: '99.8%',
   };
 };
+
 const SidePanel = ({
   side,
   className,
-  activeTabIndex: activeTabIndexProp = null,
+  activeTabIndex: activeTabIndexProp,
+  isExpanded,
   tabs,
   onOpen,
   onClose,
-  expandedWidth = 280,
   onActiveTabIndexChange,
+  expandedWidth = 280,
+  collapsedWidth = 25,
+  expandedInsideBorderSize = 4,
```

Comment:
can you add some docs what are each of these, isExpanded, expandedInsideBorderSize etc